packages/secrets/OVERVIEW.md

```diff
@@ -0,0 +1,46 @@
+# Secrets SDK
+
+Secrets for Zowe SDK is a new development package used in the following Zowe releases:
+- Zowe CLI, v7.18.0 and above
+- Zowe Explorer and Zowe Explorer API
+    - v2.10.0 and above (v2 LTS)
+    - v1.22.5 and above (v1)
+- Zowe v2.11 bundle
```

Comment:
Should we be more specific about what the bundle is and where it comes from?

Maybe we could have 2 bullet points for Zowe CLI above:
> - Zowe CLI and SDKs
>     - v7.18.0 and above from NPM package
>     - v2.11.0 and above from bundle on zowe.org

Reply:
Going to omit "and SDKs", but I like this idea. Implemented in upcoming commit